.gitignore

```diff
@@ -1,9 +1,14 @@
 bin/*
-configs/keystore/*
-!configs/keystore/.gitkeep
-configs/collector.env
-configs/password.secret
-
 .DS_Store
-celo.dockercmd
-*.json
+
+build/*-TEST-LOCAL
+build/attestation-node/*
```

Comment:
i think you can clean all this up with something like 

!**/node.attestation-service.json
!**/node.validator.json
...... ect